src/languages/pf.js

```diff
@@ -1,5 +1,5 @@
 /*
-Language: packet filter config
```

Comment:
Is packet filter an actual proper noun here?

Reply:
Looks proper to be written as PF alone maybe?

<https://en.wikipedia.org/wiki/PF_(firewall)>

Reply:
From that article it looks like your change here is fine.  Resolving this.